### PR TITLE
feat : add general information view on project details

### PR DESCRIPTION
**Pull request by gca-axelor**
RM#78171

### CODE REVIEW

**Comment by gca-axelor** [comment]
need rebase when #547 is merge

File: packages/apps/project/src/components/molecules/PartnerIconCard/PartnerIconCard.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        partnerPicture={partnerPicture}
+        partnerJob={partnerJob}
+      />
+      <View style={styles.iconContainer}>
+        <CardIconButton
+          iconName={isContact ? 'telephone-fill' : 'geo-alt-fill'}
+          iconColor={Colors.secondaryColor_dark.background}
+          onPress={() => {
+            isContact
+              ? linkingProvider.openCallApp(fixedPhone || mobilePhone)
```
Comment (vhu-axelor) [inline packages/apps/project/src/components/molecules/PartnerIconCard/PartnerIconCard.tsx:66]: Inverse mobilePhone and fixedPhone order

File: packages/apps/project/src/components/molecules/PartnerIconCard/PartnerIconCard.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import {linkingProvider} from '@axelor/aos-mobile-core';
+import {PartnerCard} from '../../atoms';
+
+interface PartnerIconCardProps {
+  style?: any;
+  partnerPicture: any;
+  partnerName: string;
+  partnerJob?: string;
+  isContact?: boolean;
+  mainAddress?: any;
```
Comment (vhu-axelor) [inline packages/apps/project/src/components/molecules/PartnerIconCard/PartnerIconCard.tsx:31]: Maybe ask fullName of mainAddress to use it directly as you do for other fields

File: packages/apps/project/src/components/molecules/ProjectHeader/ProjectHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+const ProjectHeader = ({project}: ProjectHeaderProps) => {
+  const Colors = useThemeColor();
+  const I18n = useTranslator();
+  const {getItemColorFromIndex} = useTypeHelpers();
+
+  const {base: baseConfig} = useSelector(state => state.appConfig);
+  const {user} = useSelector(state => state.user);
+
```
Comment (vhu-axelor) [inline packages/apps/project/src/components/molecules/ProjectHeader/ProjectHeader.tsx:45]: Remove space

File: packages/apps/project/src/components/molecules/ProjectHeader/ProjectHeader.tsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+  const {projectStatusList} = useSelector(state => state.project_project);
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.rowContainerGlobal}>
```
Comment (vhu-axelor) [inline packages/apps/project/src/components/molecules/ProjectHeader/ProjectHeader.tsx:50]: Remove this view and put style on container

File: packages/apps/project/src/components/molecules/ProjectHeader/ProjectHeader.tsx [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+        <View style={[styles.columnContainer, styles.flexShrink]}>
```
Comment (vhu-axelor) [inline packages/apps/project/src/components/molecules/ProjectHeader/ProjectHeader.tsx:51]: Remove flexShrink style, not useful here

File: packages/apps/project/src/components/molecules/ProjectHeader/ProjectHeader.tsx [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+  rowContainerGlobal: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'flex-start',
+  },
+  columnContainer: {
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+    marginBottom: 3,
+  },
```
Comment (vhu-axelor) [inline packages/apps/project/src/components/molecules/ProjectHeader/ProjectHeader.tsx:112]: Remove alignItems on both style, not useful

File: packages/apps/project/src/components/pages/GeneralInformationView/GeneralInformationView.tsx [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  scrollView: {
+    height: null,
+  },
+  margin: {
+    marginHorizontal: 24,
```
Comment (vhu-axelor) [inline packages/apps/project/src/components/pages/GeneralInformationView/GeneralInformationView.tsx:149]: `width: '90%', alignSelf: 'center',` to harmonize with the other components

File: packages/apps/project/src/components/atoms/PartnerCard/PartnerCard.tsx [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+          items: [
+            {
+              displayText: partnerName,
+              isTitle: true,
+            },
+            {
+              displayText: partnerJob,
+              iconName: 'suitcase-lg-fill',
+              hideIfNull: true,
+            },
```
[the remark this answers is not in the document]
Comment (lme-axelor) [reply packages/apps/project/src/components/atoms/PartnerCard/PartnerCard.tsx:59]: use indicatorText instead

File: packages/apps/project/src/components/pages/GeneralInformationView/GeneralInformationView.tsx [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+          </View>
+          {baseConfig?.enableSiteManagementForProject && (
+            <TagList title={I18n.t('Project_Sites')} tags={siteSet} />
+          )}
+        </View>
+        <NotesCard
+          title={I18n.t('Base_Description')}
+          data={project?.description}
+          style={styles.notesCard}
+        />
```
[the remark this answers is not in the document]
Comment (lme-axelor) [reply packages/apps/project/src/components/pages/GeneralInformationView/GeneralInformationView.tsx:101]: If it's due to the component, create another ticket in dev

**Pull request merged**